src/views/AnalysisModule/index.tsx

```diff
@@ -181,12 +198,11 @@ function AnalysisModule(props: AnalysisModuleProps) {
                 />
             </Container>
             {showAnaylsisAddModal && (
```

Comment:
What is showAnaylsisAddModal ?

Reply:
This is a local state variable to show analysis add modal.

Reply:
I meant the typo, but okay!